docs/esm_variables.rst

```diff
@@ -0,0 +1,120 @@
+
+  * **Setup/model elements**: `YAML-scalars`, `lists` of `dictionaries` that contain
+    information defined in the model/setup config files (i.e. ``awicm.yaml``, ``fesom.yaml``, etc.).
+    This information is model/setup-specific and causes no effect unless it is combined with the
+    **tools-specific elements**. For example, in ``fesom.yaml`` for `FESOM-1.0` the variable
+    ``asforcing`` exists, however this means nothing to `ESM-Tools` by its own. In this case, this
+    variable is used in ``namelist_changes`` (a tool-specific element) to state the type of forcing
+    to be used and this is what actually makes a difference to the simulation. The advantage of
+    having this variable already defined and called in ``namelist_changes``, in the ``fesom.yaml``
+    is that the front-end user can simply change the forcing type by changing the value of
+    ``asforcing`` (no need for the front-end user to use ``namelist_changes``).
+
+The following subsection lists and describes the **Tool-specific elements** used to operate `ESM-Tools`.
+
+.. Note::
+   Most of the **Tool-specific elements** can be defined in any file (i.e. `configuration file`,
+   `runscript`, ...) and, if present in two files used by ESM-Tools at a time, the value is chosen
+   depending on the ESM-Tools file priority/read order (:ref:`yaml_hierarchy:YAML File Hierarchy`).
+   Ideally, you would like to declare as many elements as possible inside the `configuration files`,
+   to be used by default, and change them in the `runscripts` when necessary. However, it is ultimately
+   up to the user where to setup the Tool-specific elements.
+
+Tool-Specific Elements/Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following keys should/can be provided inside configuration files for models
+(``<PATH>/esm_tools/configs/components/<name>/<name>.yaml``), coupled setups
+(``<PATH>/esm_tools/configs/setups/<name>/<name>.yaml``) and runscripts. You can find
+runscript templates in ``esm_tools/runscripts/templates/``).
+
+Installation variables
+----------------------
+
+.. csv-table::
+   :header: Key, Description
+   :widths: 15, 85
+
+   model,               "Name of the model/setup as listed in the config files (``esm_tools/configs/components`` for models and ``esm_tools/configs/setups`` for setups)."
+   setup_name,          Name of the coupled setup.
+   version,             "Version of the model/setup (one of the available options in the ``available_versions`` list)."
+   available_versions,  List of supported versions of the component or coupled setup.
+   git-repository,      Address of the model's git repository.
+   branch,              Branch from where to clone.
+   destination,         "Name of the folder where the model is download and compiled, in a coupled setup."
+   comp_command,        Command used to compile the component.
+   executable,          Name of the component executable file.
```

Comment:
I could imagine that it is unclear, wether a user can set/choose the name of the executable or just need to put here the name of executable, that the component gets through compilation. I think the latter case is true.